.github/workflows/release-container.yml

```diff
@@ -0,0 +1,47 @@
+  name: Release Keycloak container
+  on:
+    workflow_dispatch:
+    push:
+      tags:
+        - '[0-9]+.[0-9]+.[0-9]+'
+  concurrency: keycloak-container-release
+
+  jobs:
+    publish:
+      if: github.repository == 'keycloak/keycloak'
+      runs-on: ubuntu-latest
+      steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
Just a nitpick. I think it was you, @stianst, who suggested to use SHAs when pointing to a 3rd party GHA as a version tag can be floating. 🙂

Reply:
I do believe that `docker` is a "trusted enough" 3rd party 🙂